dashr/chapters/dashDataTable/index.R

```diff
@@ -74,6 +93,163 @@ layout <- htmlDiv(
                     ")
       )
-    )
+    ),
+    
+    htmlDiv(
+      list(
+        subTitleLink("Part 2", "Styling"),
+        dccMarkdown("The style of the DataTable is highly customizable. This chapter includes examples for:
+
+- Conditional formatting
+
+- Displaying multiple rows of headers
+
+- Highlighting rows, columns, and cells
+
+- Styling the table as a list view
+
+- Changing the colors (including a dark theme!)
+
+The sizing API for the table has been particularly tricky for us to nail down, so be sure to read this chapter to understand the nuances, 
+limitations, and the APIs that we're exploring.
+                    ")
+      )
+    ),
+    
+    htmlDiv(
+      list(
+        subTitleLink("Part 3", "Sorting, Filtering, Selecting and Paging"),
+        dccMarkdown("The DataTable is interactive. 
+This chapter demonstrates the interactive features of the table and 
+how to wire up these interations to R callbacks. 
+These actions include:
+                    
+- Paging
+                    
+- Selecting Rows
+                    
+- Sorting Columns
+                    
+- Filtering Data")
+      )
+    ),
+    
+    htmlDiv(
+      list(
+        subTitleLink("Part 4", "Sorting, Filtering, Selecting and Paging with R"),
+        dccMarkdown("
+In Part 3, the paging, sorting, and filtering was done entirely clientside (in the browser). 
+This means that you need to load all of the data into the table up-front. If your data is large, then this can be prohibitively slow.
+
+In this chapter, you'll learn how to write your own filtering, sorting, and paging backends in R with Dash. 
+We'll do the data processing with `dplyr` but you could write your own routines with SQL or even generate the data on the fly!
+                    ")
+      )
+    ),
+    
+    htmlDiv(
+      list(
+        subTitleLink("Part 5", "Typing"),
+        dccMarkdown("
+In this chapter, you'll learn how to configure the table to
+
+- assign the column type
+
+- change the data presentation
+
+- change the data formatting
+
+- validate or coerce user data input
+
+- apply default behavior for valid and invalid data
+                    ")
+      )
+    ),
+    
+    htmlDiv(
+      list(
+        subTitleLink("Part 6", "Editable Tables"),
+        dccMarkdown("
+The DataTable is editable. Like a spreadsheet, 
+it can be used as an input for controlling models with a variable number of inputs.
+
+This chapter includes recipes for:
+
+- Determining which cell has changed
+
+- Filtering out null values
+
+- Adding or removing columns
+
+- Adding or removing rows
+
+- Ensuring that a minimum set of rows are visible
+
+- Running computations on certain columns or cells
+                    ")
+      )
+    ),
+    
+    htmlDiv(
+      list(
+        subTitleLink("Part 7", "Rendering Cells as Dropdowns"),
+        dccMarkdown("
+Cells can be rendered as editable Dropdowns. 
+This is our first stake in bringing a full typing system to the table. 
```

Comment:
```suggestion
This is our first step in bringing a full typing system to the table. 
```
(Or maybe "stake" is correct here -- not sure if you meant it that way!)

Reply:
I think step is better